docs/backend/guides/access-global-variables.md

```diff
@@ -29,10 +29,10 @@ We use command line flags to get the environment variables. The flags are set in
 
 To add an application variable to the handler config, we create essentially a getter and setter in the handler config.
 (Ex. `SetUseSecureCookie` and `UseSecureCookie`)
-Follow the pattern in [pkg/handlers/contexts.go](https://github.com/transcom/mymove/blob/master/pkg/handlers/contexts.go)
+Follow the pattern in [pkg/handlers/config.go](https://github.com/transcom/mymove/blob/main/pkg/handlers/config.go)
```

Comment:
Could someone verify that this the correct file to link to? I think this what `contexts.go` got moved/renamed to.

Reply:
Yes exactly correct. It was renamed in this commit ae1c895d871099f6fc4fe3ee8805892602f0e169. You check out the bottom of this page for proof.

[=> this page shows the history of the file `config.go`](https://github.com/transcom/mymove/commits/main/pkg/handlers/config.go)